src/posts/2020-06-08-going-remote.md

```diff
@@ -0,0 +1,110 @@
+
+But when we made the switch, it was hard to get everyone on board with it and to add a tool to the workflow. It was just *too out of hand*. Now, to me, this is the greatest advantage of Basecamp. Even more so if you disable the notifications. You actually have to set time aside to check and devote to it, just like you would for e-mail. This is wonderful for asynchronous discussions since you'll have a period devoted to thinking them through and writing coherent and structured thoughts.
+
+However, to add that one tool that is *out of hand*, that requires you to **consciously make an effort of using it and checking it**, might be a difficult habit to create. In the beginning, we were seeing a lot of discussions with barely any answers, or none at all. You would scroll around in message boards of different topics, see people asking for opinions and getting 1, 2, maybe 3 others to pitch in. That's 25-33% of our company. That wasn't enough. Besides Basecamp being out of hand, a big factor that lead to it was what I mentioned previously - a lot of discussions were happening offline and people weren't accostumed to sharing the outcomes, even though Basecamp is perfect for that. The lack of responses was leading to a negative feedback loop: nobody replies, poster gets discouraged, fewer posts, fewer replies. If you're remote and the rest of your team is in an office, this is a big factor in feeling cut-off from the decision making progress.
+
+Eventually, we made a conscious effort. **Started sending out reminders on Slack, asking for opinion** and tagging directly and some (*cough Roberto cough*) were the **acting police officers replying "yeah that's cool and all but can you move it to Basecamp?" in every. single. discussion. on Slack**. For me, this was a game changer and thanks to the efforts in making the switch, our communication and decision making process improved dramatically.
+
+### Meetings
+
+*Beware! Here be dragons!*
+
+Whenever someone is enthusiastic about "*more meetings!*", you should be suspicious. Especially under the guise of "fighting remote isolation". And this happens because of two reasons. The first one is that as a company you want to maximise productivity and **nothing is a better killer of productivity than unnecessary meetings with too many people**. As put so elloquently by Jason Fried and DHH in their *Remote: Office not Required* book:
+
+> Pulling seven people away from their work for an hour is worth seven hours of lost productivity
```

Comment:
Our quotes are styled so that the last paragraph is the name of the author, so I recommend going for it instead of using that last sentence in the previous paragraph.